lib/controller.js

```diff
@@ -45,152 +33,134 @@ const getEmoji = (repoLength) => {
     return '';
 };
 
-// handle search and display statistics
-exports.index = (req, res) => {
-    const username = req.query.username;
-    const headers = {
-        'User-Agent': `${username}`
-    };
-
-    const userOptions = {
-        url: `https://api.github.com/users/${username}?client_id=${client_id}&client_secret=${client_secret}`,
-        headers
-    };
-
-    const repoOptions = {
-        url: `https://api.github.com/users/${username}/repos?client_id=${client_id}&client_secret=${client_secret}&per_page=100`,
-        headers
-    };
+const reqUser = (username) => {
+    const url = `https://api.github.com/users/${username}?client_id=${client_id}&client_secret=${client_secret}`;
+    const headers = { 'User-Agent': `${username}` };
```

Comment:
Why do you re-declare the `headers` twice? in `reqUser` and `reqRepos`?

Reply:
just to avoid having to pass them as params... but it can be declared outside both and passed by params too

Reply:
what about this one @wildan3105? better as param or declared inside?

Reply:
Okay then, just leave it as it is, no need to change :ok_hand:

Reply:
Which part do you mean @juancuiule ?

Reply:
The last change you made, it's good. So we will stick with it about `headers` declaration. Nice.